pkg/steps/multi_stage.go

```diff
@@ -158,6 +171,54 @@ func (s *multiStageTestStep) Provides() (api.ParameterMap, api.StepLink) {
 }
 func (s *multiStageTestStep) SubTests() []*junit.TestCase { return s.subTests }
 
+func (s *multiStageTestStep) setupRBAC() error {
+	labels := map[string]string{multiStageTestLabel: s.name}
+	m := meta.ObjectMeta{Name: s.name, Labels: labels}
```

Comment:
There is the potential for conflict here: the test cannot be named `default`, `ci-operator`, `default`, or `deployer`. These are all `create` calls and the test will fail with an obvious message, so I don't think it is a significant problem.